src/drivers/vue.js

```diff
@@ -28,7 +28,12 @@ type VueType = {|
 function propsToCamelCase(props : Object) : Object {
     return Object.keys(props).reduce((acc, key) => {
         const value = props[key];
-        if (key.includes('-')) {
+        // In vue `style` is a reserved prop name
+        if (key === 'style-object' || key === 'styleObject') {
+            acc.style = value;
+            // To keep zoid as generic as possible, passing in the original prop name as well
+            acc[key] = value;
```

Comment:
If key is indeed `style-object`, this would not convert it to camel case, right?
Do we want that? If not, using `acc.styleObject = value` would force the camel case even on this scenario.

Reply:
@andregardi you are right, here is better just set:

```
acc.styleObject = value;
```

I have done the change.

Reply:
Great! Thank you for that!